packages/gatsby-dev-cli/src/watch.js

```diff
@@ -31,6 +31,7 @@ function watch(root, packages, { scanOnce, quiet }) {
     const ignoreRegs = [
       /[/\\]node_modules[/\\]/i,
       /\.git/i,
+      /\.DS_Store/,
```

Comment:
I was seeing some `.DS_Store` files coming over from packages/ so added to the ignore list